packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -252,9 +278,17 @@ export const NavigationDrawerItem = ({
             />
           )}
 
-          <NavigationDrawerAnimatedCollapseWrapper>
-            <StyledItemLabel>{label}</StyledItemLabel>
-          </NavigationDrawerAnimatedCollapseWrapper>
+          <StyledLabelParent>
```

Comment:
where is the `<NavigationDrawerAnimatedCollapseWrapper`?

Reply:
we need it for collapse animation

Reply:
@martmull That's the issue. Using `NavigationDrawerAnimatedCollapseWrapper` won't let the text truncate for any item because a width constraint is needed for the parent element to make it work. 
Also, you can find `StyledRightOptionsContainer` having `margin-left: auto`, but it doesn't work due to the `NavigationDrawerAnimatedCollapseWrapper`, we can see it working due to the `StyledSpacer`. 

I will see how I can make changes to the `NavigationDrawerAnimatedCollapseWrapper` to make flex property work.